TYPO3.Flow/Classes/TYPO3/Flow/Object/CompileTimeObjectManager.php

```diff
@@ -339,36 +340,50 @@ function ($className) use ($filterExpression, $includeOrExclude) {
     protected function buildObjectsArray()
     {
         $objects = array();
+        /** @var Configuration $objectConfiguration */
         foreach ($this->objectConfigurations as $objectConfiguration) {
```

Comment:
The diff looks more complex than it is: I just extracted the loop as is to a new `buildObjectArray()` method so it can be called for the main object and it's presets